templates/libvirtd.conf

```diff
@@ -0,0 +1,20 @@
+#
+# This file is managed with puppet (class ::libvirt::config)
+#    do not edit manually.
+#
+# Master configuration file for the libvirtd service.
+# All settings described here are optional - if omitted, sensible
+# defaults are used.
+#
+# See https://github.com/libvirt/libvirt/blob/master/src/remote/libvirtd.conf
+# for possible values.
+<%- @libvirtd_conf.each do |key,value| -%>
+    <%- if ( value.is_a? Array ) -%>
+<%= key %> = ["<%= value.join('", "') %>"]
+   <%- elsif  (( Integer(value) and Integer(value) <= 9) rescue false) -%>
```

Comment:
I definitly do not like that.
why not leave it like in qemu ?
if you thean have:
var => 100    it's an integer and becomes var=100
var => '100'  it's a string and becomes var='100'
I do not believe that integer above 10 are alway integers and above always strings (eg. max_clients is a integer !)